undead-discord.css

```diff
@@ -33,6 +33,64 @@
   border-bottom: 1px solid var(--darkest-bg-color);
 }
 
+/* The following 3 blocks color the keybind-shortcut modal and buttons */
+.ddr-arrows { display: none; }
+
+.keybind-shortcut > span {
+    color: #4F545C !important;
```

Comment:
I don't want colors set to specific values anywhere. If this seems necessary, please use the fg and secondary fg colors.

Reply:
Can't remember why I did this, but I'll do something about it

Reply:
I take that back, yes I do. It's because Discord has hard-coded the color of the arrows in the keybind boxes. This was to match the font color in each keybind box with the arrows as well, otherwise, looks like this:

![keybind_colors](https://user-images.githubusercontent.com/647577/33100290-d730b602-cee1-11e7-89f1-75716b6ecf40.png)

Reply:
Fixed, turns out you can edit svg fill with css, so I did that